source/material_model/visco_plastic.cc

```diff
@@ -111,11 +111,11 @@ namespace aspect
       /* The following returns whether or not the material is plastically yielding
        * as documented in evaluate.
        */
-      const std::pair<std::vector<double>, std::vector<bool>> calculate_viscosities =
+      const IsostrainViscosities isostrain_viscosities =
```

Comment:
The indentation check currently fails here. This can now go on one line.

Reply:
it is a really long line now, should I break it into several?

Reply:
It's still shorter than it used to be, so I think this is ok. The last variable has a very long name anyway, so breaking the line up wouldn't help all that much.